src/App.js

```diff
@@ -7,7 +7,11 @@ function App() {
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
```

Comment:
- [ ] When I run your app I get this error message which prevents the app from running. Kindly fix this. You need to add back the deleted `src/logo.svg` file or remove its use in this file.

  ![image](https://user-images.githubusercontent.com/54848363/220176951-0aec0ee9-f22f-4820-b489-cdd95b4f2750.png)

Reply:
@DammyShittu Alright. I got that.

Reply:
@DammyShittu Done.

Reply:
- Well resolved @chepkok3 
<img width="843" alt="Screenshot 2023-02-20 at 19 34 06" src="https://user-images.githubusercontent.com/58692590/220178973-951c2ca4-7ac3-4b9f-8bfe-a55737410c99.png">

Reply:
@Whoistolu 👍🏿